java/client/src/org/openqa/selenium/remote/http/netty/NettyHttpHandler.java

```diff
@@ -53,7 +55,11 @@ private HttpResponse makeCall(HttpRequest request) {
     Require.nonNull("Request", request);
 
     Future<Response> whenResponse = client.executeRequest(
-      NettyMessages.toNettyRequest(getConfig().baseUri(), request));
+      NettyMessages.toNettyRequest(
+        getConfig().baseUri(),
+        toClampedInt(getConfig().readTimeout().toMillis()),
+        toClampedInt(getConfig().readTimeout().toMillis()),
```

Comment:
One down side of this is that we cannot add a `connectionTimeout` per request. Currently the default is 10s, we could leave a high value (60s?) to fit most use cases.
What do you think, @shs96c @barancev?